docs/source/getting_started/SettingsEditor.rst

```diff
@@ -0,0 +1,809 @@
+    mpedit = MpConfigEdit()
+    mpedit
+
+
+You should see your saved settings displayed.
+
+
+More Settings
+-------------
+
+Adding Threat Intelligence (TI) Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will likely want to do lookups of IP Addresses, URLs and other items
+to check for any Threat Intelligence reports. To do that you need to add
+the providers that you want to use. Most TI providers require that you
+have an account with them and supply an API key or other authentication
+items when you connect.
+
+Most providers have a free use tier (or in cases like AlienVault OTX)
+are entirely free. Free tiers for paid providers usually impose a
+certain number of requests that you can make in a given time period.
+
+For account creation, each provider does this slightly differently. Use
+the help links in the editor help to find where to go set each of these
+up.
+
+Assuming that you have done this, we can configure a provider. Be sure
+to store any authentication keys somewhere safe (and memorable).
+
+We are going to use `VirusTotal <https://www.virustotal.com>`__ (VT) as
+an example TI Provider. For this you will need a VirusTotal API key from
+the
+`VirusTotal <https://developers.virustotal.com/v3.0/reference#getting-started>`__
+website. We also support a range of other threat intelligence providers
+- you can read about this here `MSTICPy
+TIProviders <https://msticpy.readthedocs.io/en/latest/data_acquisition/TIProviders.html>`__
+
+Taking VirusTotal as our example.
+
+- Click on the **TI Providers** tab
+- Select "VirusTotal" from the **New prov** drop-down list
+- Click **Add**
+
+This should show you the values that you need to provide:
+
+- a single item **AuthKey** (this is usually referred to as an “API Key”)
+
+You can paste the key into the **Value** field and click the **Save**
+button. Do not surround the value with quotes.
+
+Instead of keeping your key in the configuration file,
+you can opt to store the VT AuthKey as an environment variable. This is
+a bit more secure than having it laying around in configuration files.
+Assuming that you have set you VT key as an environment variable
+
+.. code:: bash
+
+   set VT_KEY=VGhpcyBzaG91bGQgc2hvdyB5b3UgdGhlIHZhbHVlcyB  (Windows)
+   export VT_KEY=VGhpcyBzaG91bGQgc2hvdyB5b3UgdGhlIHZhbHVlcyB  (Linux/Mac)
+
+Flip the **Storage** radio button to **EnvironmentVar** and type the
+name of the variable (``VT_KEY`` in our example) into the value box.
+
+You can also use Azure Key Vault to store secrets like these but we will
+need to set up the Key Vault settings before this will work. This
+is covered later in `Key Vault Secrets`_ and `Key Vault`_.
+
+Click the **Save File** button to save your changes.
+
+
+.. figure:: _static/settings_ti_provs.png
+   :alt: Threat intelligence provider settings for VirusTotal
+
+
+Test that the TI settings work
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This assumes that you have set an enviroment variable pointed at your
```

Comment:
environment

Reply:
I missed this but have added it to a current branch.